anesthetic/plot.py

```diff
@@ -450,15 +464,22 @@ def fastkde_contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
         A set of contourlines or filled regions
 
     """
+    kwargs = normalize_kwargs(kwargs, dict(linewidths=['linewidth', 'lw'],
+                                           linestyles=['linestyle', 'ls'],
+                                           color=['c'],
+                                           facecolor=['fc'],
+                                           edgecolor=['ec']))
     xmin = kwargs.pop('xmin', None)
     xmax = kwargs.pop('xmax', None)
     ymin = kwargs.pop('ymin', None)
     ymax = kwargs.pop('ymax', None)
     label = kwargs.pop('label', None)
     zorder = kwargs.pop('zorder', 1)
-    linewidths = kwargs.pop('linewidths', 0.5)
     levels = kwargs.pop('levels', [0.68, 0.95])
     color = kwargs.pop('color', next(ax._get_lines.prop_cycler)['color'])
+    facecolor = kwargs.pop('facecolor', color)
+    edgecolor = kwargs.pop(
+        'edgecolor', color if facecolor in [None, 'None', 'none'] else 'k')
```

Comment:
The matplotlib standard for markers or similar is to use `'None'` or `'none'` to make them unfilled, because `None` typically is used for automatic color assignment. This is a common gotcha for many matplotlib users, and since we don't have any use for `None` here, I've added it to the list of values requesting unfilled contours.